### PR TITLE
Dev adfs

### PR DESCRIPTION
**Pull request by nessiLahav**
#### What does this PR do?
Configuring ADFS(static env) as OpenID provider for manual testing
#### What ticket does this PR close?
 connected to #860 
#### Where should the reviewer start?
In start script
#### How should this be manually tested?
Run `./start —authn-oidc —oidc-adfs`
#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/dev_adfs/2/

### CODE REVIEW

File: ci/authn-oidc/policy.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@
```
Comment (orenbm) [inline ci/authn-oidc/policy.yml]: This file is getting too big. Can we separate this file to `okta-policy.yml`, `keycloak-policy.yml` & `adfs-policy.yml`?
Then we can load the different policies in each section per oidc provider

File: dev/start [hunk 2]
```diff
@@ -15,7 +15,9 @@ Usage: start [options]
 
     --authn-iam     Starts with authn-iam/prod as authenticator
 
-    --authn-oidc    Starts with authn-oidc/okta as authenticator
+    --authn-oidc    Starts with authn-oidc/keycloak as authenticator
+
+    --oidc-adfs     Adds to authn-oidc adfs static env configuration
```
Comment (orenbm) [inline dev/start]: What happens if we run `./start --oidc-adfs` without `--authn-oidc`? Is this the behaviour we want?

File: dev/start [hunk 3]
```diff
@@ -83,11 +87,18 @@ if [[ $ENABLE_AUTHN_OIDC = true ]]; then
   echo "Configuring Keycloak as OpenID provider for automatic testing"
   docker-compose exec conjur conjurctl policy load cucumber /src/conjur-server/ci/authn-oidc/policy.yml
 
-  enabled_authenticators="$enabled_authenticators,authn-oidc/okta,authn-oidc/keycloak"
+  enabled_authenticators="$enabled_authenticators,authn-oidc/okta,authn-oidc/keycloak,authn-oidc/adfs"
```
Comment (orenbm) [inline dev/start]: The addition of `authn-oidc/adfs` can actually move inside the if clause below

File: dev/start [hunk 4]
```diff
@@ -114,6 +125,10 @@ if [[ $ENABLE_AUTHN_OIDC = true ]]; then
   docker-compose exec client conjur variable values add conjur/authn-oidc/okta/provider-uri https://dev-842018.oktapreview.com
   docker-compose exec client conjur variable values add conjur/authn-oidc/okta/id-token-user-property "preferred_username"
 
+  # add variables' values for ADFS
+  docker-compose exec client conjur variable values add conjur/authn-oidc/adfs/provider-uri https://adfs4win2016.northeurope.cloudapp.azure.com
+  docker-compose exec client conjur variable values add conjur/authn-oidc/adfs/id-token-user-property "unique_name"
```
Comment (orenbm) [inline dev/start]: this should go inside the if clause

File: dev/start (hunk 2 above)
Comment (nessiLahav) [reply dev/start]: Fix that :)

File: ci/authn-oidc/policy.yml (hunk 1 above)
Comment (nessiLahav) [reply ci/authn-oidc/policy.yml]: Done :)

File: dev/start (hunk 3 above)
Comment (nessiLahav) [reply dev/start]: Done

File: dev/start (hunk 4 above)
Comment (nessiLahav) [reply dev/start]: Done

**Pull request merged**